### DISCUSSION TITLE
Video Player constantly crashes

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/security-and-auditing-full-course-s23#41] [General]
1. The Video player crashes often. I attach a screenshot of the mistake
<img width="648" alt="Screenshot 2023-12-22 at 1 18 22" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/23662270/dbce05cd-6703-41c0-9895-ca86014bbd09">

2. Shrinking back the player from fullscreen is not possible from the button in the bottom right corner. Escape has to be used

**Comment by PatrickAlphaC** [comment]
Thanks! Which chapter is this on?

**Reply by ghost** [reply]
I do not remember, on which chapter I took the screenshot. However it happens randomly and the issue is not bound to a specific chapter. I am just informing you about this common bug. I guess it is the player that you are using, which causes the problem and it might not be easily fixable, due to the third party libraries you are using. I imagine it happens to other people too. Good luck with the course.

**Reply by cromewar** [reply]
@vtonchev already informed to the dev team we will try to have this solved asap @vtonchev, thanks!
